php/post-types/class-block-post.php

```diff
@@ -578,7 +620,15 @@ public function render_fields_meta_box_row( $field, $uid = false ) {
 								id="block-fields-edit-label-input_<?php echo esc_attr( $uid ); ?>"
 								class="regular-text"
 								value="<?php echo esc_attr( $field->label ); ?>"
-								data-sync="block-fields-label_<?php echo esc_attr( $uid ); ?>" />
+								data-sync="block-fields-label_<?php echo esc_attr( $uid ); ?>"
+								<?php echo $is_field_disabled ? 'readonly="readonly"' : ''; ?> />
+							<?php if ( $is_field_disabled ) : ?>
+								<input
+									name="block-is-disabled-pro-field[<?php echo esc_attr( $uid ); ?>]"
+									type="hidden"
```

Comment:
Having a hidden `<input>`to show that the field is disabled isn't idea. 
 
But because I set the `<select>` for the control type as `disabled`, its value won't be present in the POST request. So I couldn't figure out how to find if it's a pro control, without adding this hidden `<input>`.

Reply:
Have you tested this out with checkboxes?